content/docs/started/k8s/kfctl-existing-arrikto.md

```diff
@@ -22,134 +22,7 @@ Read the relevant [article](https://journal.arrikto.com/kubeflow-authentication-
-    apiVersion: v1
-    kind: ConfigMap
-    metadata:
-      namespace: metallb-system
-      name: config
-    data:
-      config: |
-        address-pools:
-        - name: default
-          protocol: layer2
-          addresses:
-          - 10.0.0.100-10.0.0.110
-    EOF
-    ```
-
-
-**Ensure that MetalLB works as expected (optional):**
-
-1. Create a dummy service:
-
-    ```
-    kubectl create service loadbalancer nginx --tcp=80:80
-    service/nginx created
-    ```
-
-1. Ensure that MetalLB has allocated an IP address for the service:
-
-    ```
-    kubectl describe service nginx
-    ...
-    Events:
-      Type    Reason       Age   From                Message
-      ----    ------       ----  ----                -------
-      Normal  IPAllocated  69s   metallb-controller  Assigned IP "10.0.0.101"
-    ```
-
-1. Check the corresponding MetalLB logs:
-
-    ```
-    kubectl logs -n metallb-system -l component=controller
-    ...
-    {"caller":"service.go:98","event":"ipAllocated","ip":"10.0.0.101","msg":"IP address assigned by controller","service":"default/nginx","ts":"2019-08-09T15:12:09.376779263Z"}
-    ```
-    
-1. Create a pod that will be exposed with the service:
-
-    ```
-    kubectl run nginx --image nginx --restart=Never -l app=nginx
-    pod/nginx created
-    ```
-    
-1. Ensure that MetalLB has assigned a node to announce the allocated IP address:
-
-    ```
-    kubectl describe service nginx
-    ...
-    Events:
-      Type    Reason       Age   From                Message
-      ----    ------       ----  ----                -------
-       Normal  nodeAssigned  4s    metallb-speaker     announcing from node "node-2"
-    ```
-    
-1. Check the corresponding MetalLB logs:
-
-    ```
-    kubectl logs -n metallb-system -l component=speaker
-    ...
-    {"caller":"main.go:246","event":"serviceAnnounced","ip":"10.0.0.101","msg":"service has IP, announcing","pool":"default","protocol":"layer2","service":"default/nginx","ts":"2019-08-09T15:14:02.433876894Z"}
-    ```
-    
-1. Check that MetalLB responds to ARP requests for the allocated IP address:
-
-    ```
-    arping -I eth0 10.0.0.101
-    ...
-    ARPING 10.0.0.101 from 10.0.0.204 eth0
-    Unicast reply from 10.0.0.101 [6A:13:5A:D2:65:CB]  2.619ms
-    ```
-    
-1. Check the corresponding MetalLB logs:
-
-    ```
-    kubectl logs -n metallb-system -l component=speaker
-    ...
-    {"caller":"arp.go:102","interface":"eth0,"ip":"10.0.0.101","msg":"got ARP request for service IP, sending response","responseMAC":"6a:13:5a:d2:65:cb","senderIP":"10.0.0.204","senderMAC":"9a:1f:7c:95:ca:dc","ts":"2019-08-09T15:14:52.912056021Z"}
-    ```
-    
-1. Verify that everything works as expected:
-
-    ```
-    curl http://10.0.0.101
-    ...
-    <p><em>Thank you for using nginx.</em></p>
-    ...
-    ```
-    
-1. Clean up:
-
-    ```
-    kubectl delete service nginx
-    kubectl delete pod nginx
-    ```
-    
-</details>
-
+* Existing Kubernetes Cluster.
```

Comment:
This line is just a bullet point in the rendered page. We need to add more context.
https://deploy-preview-1265--competent-brattain-de2d6d.netlify.com/docs/started/k8s/kfctl-existing-arrikto/#before-you-start

Reply:
Ah, I think I see now what the line means. Does this fit the intended meaning?

`The instructions below assume that you have an existing Kubernetes cluster.`

*No bullet point needed, just the sentence.*

Reply:
Much better, will fix.